docs/spec.md

```diff
@@ -446,18 +435,19 @@ to that of the derivation (or derivations).
 
 
 
-## Traversable
-
-#### Dependencies
-
-  1. Functor
-  1. Foldable
+### Traversable
 
-#### Methods
+In the following signature `Applicative<U>` means that a value must
+not only match `Applicative` signature, but also fully support `Applicative` algebra.
 
-  1. `traverse :: (Traversable t, Applicative f) => Type t ~> (Type f, (a → f b), t a) → f (t b)`
+```js
+Traversable<T> {
+  traverse: (Applicative<U>, a => U<b>, T<a>) => U<T<b>>
+}
+```
```

Comment:
Well because in notation that I made up `Applicative<U>` in that position means "a module that matches `Applicative` signature for type `U`" :)

Check out section that describes the notation.